metricbeat/docs/running-on-docker.asciidoc

```diff
@@ -19,6 +19,7 @@ docker run \
   --mount type=bind,source=/sys/fs/cgroup,target=/hostfs/sys/fs/cgroup,readonly \ <2>
   --mount type=bind,source=/,target=/hostfs,readonly \ <3>
   --net=host \ <4>
+  --user root --cap-add sys_ptrace --cap-add dac_read_search \ <5>
```

Comment:
This is not needed with default settings, I wonder if it adding these here will mislead users to always add this capabilities

Reply:
The comment _does_ mention that this is for the system/socket metricset, which is probably enough. Also, I wouldn't be surprised if other metricsets have similar permissions issues that we haven't discovered.